lib/src/main/kotlin/com/swmansion/starknet/provider/gateway/GatewayProvider.kt

```diff
@@ -444,7 +444,24 @@ class GatewayProvider(
         }
 
         @JvmStatic
-        fun makeTestnetClient(httpService: HttpService): GatewayProvider {
+        fun makeTestnetProvider(testnetId: StarknetChainId): GatewayProvider =
+            when (testnetId) {
+                StarknetChainId.TESTNET -> GatewayProvider(
+                    "$TESTNET_URL/feeder_gateway",
+                    "$TESTNET_URL/gateway",
+                    testnetId,
+                )
+//                FIXME(restore this after 0.5.0 rebase on main)
+//                StarknetChainId.TESTNET2 -> GatewayProvider(
+//                    "$TESTNET2_URL/feeder_gateway",
+//                    "$TESTNET2_URL/gateway",
+//                    testnetId,
+//                )
```

Comment:
Once milestone/0.5.0 is properly rebased (after 0.4.0 is merged) this will have to be restored.